frontend/src/screens/emailVerificationScreen.tsx

```diff
@@ -69,13 +69,14 @@ const styles = StyleSheet.create({
 
     containedContainer: {
         width: "100%",
+        height: "87%",
         alignItems: "center"
     },
 
     mainImage: {
         width: "60%",
         resizeMode: 'contain',
-        maxHeight: 158,
+        maxHeight: "26.4%",
```

Comment:
What are your thoughts on making these nice numbers, like 85, 25, and 2%? I don't think it would make too much of a difference visually, but I think it would be neater to not have decimals here.

Otherwise, it looks good!